lib/src/lis/io.cpp

```diff
@@ -420,6 +436,11 @@ record_index iodevice::index_records() noexcept (true) {
                 ex.pop_back();
                 break;
             }
+            case lis::record_type::logical_eof:
```

Comment:
I would say that logical EOF should go to the previous category - we also want it to be treated separately so it can be quietly dismissed? Unless you want to present it to the user.

On the other hand I assume they might never appear in the real world, so shouldn't matter much.